subprojects/degrees-and-programs/source/static/app/sass/helpers/mixins/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "brei-sass-mixins",
+  "version": "1.0.0",
+  "description": "BarkleyREI sass mixins",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nessthehero/brei-sass-mixins.git"
+  },
+  "author": "Ian Moffitt <nessthehero@gmail.com> (http://www.ianmoffitt.co/)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/nessthehero/brei-sass-mixins/issues"
+  },
+  "homepage": "https://github.com/nessthehero/brei-sass-mixins#readme"
+}
```

Comment:
Main subproject README needs to explain the role of this `static` directory. It looks like this is the source that's being edited by coders to build code for deployment. Maybe it's just the term "static" that's confusing me.